api/http/templates.go

```diff
@@ -74,229 +74,1311 @@ const bzzList = `{{ define "content" }}
+									C11.9,7.4,12.2,7.6,12.4,7.8L12.4,7.8z M17.1,3c-2-2-5.3-2-7.3,0L7.6,5.3c0,0,0,0,0,0c-2,2-2,5.2,0,7.2c0.2,0.2,0.4,0.4,0.7,0.6
+									c0.2,0.2,0.5,0.1,0.7-0.1l0.7-0.7c0.3-0.3,0.4-0.6,0.4-1c0-0.2-0.1-0.3-0.2-0.4c-0.1-0.1-0.2-0.2-0.3-0.3c-0.9-0.9-1-2.5,0-3.4
+									L11.8,5c1-1,2.5-0.9,3.5,0c0.9,1,0.9,2.5,0,3.4l-0.4,0.4c-0.1,0.1-0.2,0.4-0.1,0.6c0.2,0.6,0.3,1.2,0.3,1.8c0,0.5,0.6,0.7,0.9,0.4
+									l1.3-1.3C19.1,8.3,19.1,5,17.1,3L17.1,3z"/>
+								</svg>
 
-<div class="searchbar">
-  <form class="separate-block" action="javascript:goToPage();">
-    <input type="text" id="page" autofocus name="search" placeholder="Please enter an ENS name or swarm hash to retrieve ..">
-    <button class="button" type="submit" value="submit" onclick="goToPage();">Go!</button>
-  </form>
-</div>
-<div class="footer">
-<p>{{.Timestamp}}</p>
-<p>Swarm provides censorship resistant storage and communication infrastructure for a sovereign digital society</p>
-</div>
-</div>
-
-{{ end }}`
+							</button>
+						</div>
+						<div class="uploadAction">
+							<button id="uploadButtonHash" class="orangeButton fadeOut" title="Copy hash to clipboard.">
+								<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+									 viewBox="0 0 20 20" style="enable-background:new 0 0 20 20;" xml:space="preserve">
+								<path id="XMLID_2810_" d="M18.9,6.9l0.3-1.7c0.1-0.3-0.2-0.6-0.5-0.6h-3.1l0.6-3.4c0.1-0.3-0.2-0.6-0.5-0.6h-1.7
+									c-0.2,0-0.4,0.2-0.5,0.4L13,4.7H8.9l0.6-3.4C9.6,1,9.3,0.8,9,0.8H7.3c-0.2,0-0.4,0.2-0.5,0.4L6.2,4.7H3c-0.2,0-0.4,0.2-0.5,0.4
+									L2.2,6.8C2.1,7.1,2.4,7.4,2.7,7.4h3.1l-0.9,5.3H1.5c-0.2,0-0.4,0.2-0.5,0.4l-0.3,1.7c-0.1,0.3,0.2,0.6,0.5,0.6h3.1l-0.6,3.4
+									c-0.1,0.3,0.2,0.6,0.5,0.6h1.7c0.2,0,0.4-0.2,0.5-0.4L7,15.3h4.1l-0.6,3.4c-0.1,0.3,0.2,0.6,0.5,0.6h1.7c0.2,0,0.4-0.2,0.5-0.4
+									l0.6-3.6H17c0.2,0,0.4-0.2,0.5-0.4l0.3-1.7c0.1-0.3-0.2-0.6-0.5-0.6h-3.1l0.9-5.3h3.3C18.7,7.4,18.9,7.2,18.9,6.9z M11.6,12.6H7.5
+									l0.9-5.3h4.1L11.6,12.6z"/>
+								</svg>
+							</button>
+						</div>
+					</div>
+					<div id="uploadFeedbackBarsWrapper">
+						<div id="uploadFeedbackBars">
+							<div class="uploadFeedbackBar uploadFeedbackColor1" id="uploadReceivedBar"></div>
+							<div class="uploadFeedbackBar uploadFeedbackColorx hidden" id="uploadSeenBar"></div>							
+							<div class="uploadFeedbackBar uploadFeedbackColor2" id="uploadSplitBar"></div>
+							<div class="uploadFeedbackBar uploadFeedbackColor3" id="uploadStoredBar"></div>
+							<div class="uploadFeedbackBar uploadFeedbackColor4" id="uploadSentBar"></div>							
+						</div>
+						<div class="incrementLine incrementLine1"></div>
+						<div class="incrementLine incrementLine2"></div>
+						<div class="incrementLine incrementLine3"></div>
+					</div>
+					<div class="uploadFeedbackCounts">
```

Comment:
As we decided to have only one progress bar in cli, should we have the same here?